packages/frontend/src/constants.ts

```diff
@@ -47,6 +51,10 @@ export const NOTIFICATIONS = (value?: string) => ({
   EVENT_DELETED: `Событие ${value} успешно удалено`,
   EVENT_DELETE_ERROR: `Ошибка при удалении события ${value}`,
   EVENT_CREATION_ERROR: `Ошибка при создании события ${value}`,
+  VOLUNTEER_EXISTS: `Извините, волонтёр c именем ${value} в Telegram уже существует. Пожалуйста, проверьте данные или обратитесь к администратору`,
```

Comment:
This is the same issue as with Hripsime - volonteers doesn't know about app and administrators, so there shouldn't be such text in the error